### PR TITLE
Add missing environment variable to manifest

### PR DESCRIPTION
**Pull request by DavidBiddle**
#### What problem does the pull request solve?
Fixes an issue with the previous sentry PR - errors in the dev enviroment were coming through as 'local', because the `PAAS_ENVIRONMENT` variable wasn't available in the app.

#### Checklist

- [x] I've used the pull request template
- [n/a] I've linked this PR to the relevant issue (if mission work)
- [n/a] I've written unit tests for these changes (if code change)
- [n/a] I've updated the documentation in (If any documentation requires updating)
    - [n/a] README.md
    - [n/a] Elsewhere (please link)

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+    memory: 256M
+    command: bundle exec rackup -o 0.0.0.0 -p $PORT
+    services:
+      - forms-api-((PAAS_ENVIRONMENT))-db
+      - forms-api-((PAAS_ENVIRONMENT))-splunk
+    env:
+      RACK_ENV: production
+      SENTRY_DSN: ((SENTRY_DSN))
+      API_KEY: ((API_KEY))
+      PAAS_ENVIRONMENT: ((PAAS_ENVIRONMENT))
```
Comment (0atman) [inline manifest.yml:13]: is this indentation correct?

Comment (DavidBiddle) [reply manifest.yml:13]: Not sure - VS Code did it automatically based on the editorconfig file. It's usually pretty good at not changing the  meaning when it does these things, and I don't _think_ Yaml cares about the difference between:
```
a
- b
```

and

```
a
  - b
```

but I'm happy to change it back if it breaks the deploy

Comment (DavidBiddle) [reply manifest.yml:13]: I've just merged something similar in the admin and it looks like the deploy ran OK - https://github.com/alphagov/forms-runner/actions/runs/2876513134

Comment (DavidBiddle) [reply manifest.yml:13]: Going to resolve this for now, we can revisit if we spot any issues (or if other people's editors start changing it back)

**Review by aliuk2012 — APPROVED** [review]

**Pull request merged**